docs/develop/Repositories.md

```diff
@@ -1,45 +1,84 @@
 # Repositories
 
-The SDK is comprised of five repositories:
+There are 5 repositories within the ElectionGuard SDK:
 
 - [ElectionGuard Specification][election-guard-specification]
 - [ElectionGuard Python][election-guard-python-source]
 - [ElectionGuard C++][election-guard-cpp-source-code]
 - [ElectionGuard Python API][election-guard-web-api-source]
 - [ElectionGuard React UI][election-guard-ui-source]
 
+If you're looking to develop for ElectionGuard, it might help if you pick something you're familiar with. The following list acts to showcase the language to assist you in finding the right place. Each repository contains information on the best way to contribute and how to setup the development environment. Take a look and see what appeals to you.
 
-![Code][code-image]
+## <div style="display: flex; align-items: center;">ElectionGuard Specification<span style="display: flex; align-items: center; margin-left: 16px"> ![Markdown][markdown-logo] <span></div>
 
-## Diagrams
+The ElectionGuard Specification repo addresses the mathematical and conceptual underpinnings of end-to-end verifiability in a manner that comfortably accommodates post-election audits. The specification houses the core data constructs, tests, and datasets that we recommend to ensure portability and data integrity.
 
-```mermaid
-flowchart TD
-    subgraph "Repositories"
-    spec["electionguard"] -.-> |python|e-python["electionguard-python"];
-    spec["electionguard"] -.-> |C++|e-cpp["electionguard-cpp"];
-    pypi --> web-api["electionguard-api-python"];
-    e-python --> |pypi|pypi["python module: electionguard"]:::package;
-    web-api --> e-ui["electionguard-ui"];
-    e-cpp --> |nuget| nuget["ElectionGuard.Encryption"]:::package;
-    end
+The ElectionGuard site is also built from this repository using mkdocs. This can be a friendly place to start out for new developers trying to get started.
 
-    subgraph "Overview"
-    specification -.-> |python|python["core"];
-    specification -.-> |C++|cpp["encryption"];
-    python --> |pypi|module:::package
-    module --> api["web-api"];
-    api --> ui["user interface"];
-    cpp --> |nuget|package:::package
-    end
+[:fontawesome-brands-github: ElectionGuard Specification][election-guard-spec-overview]
 
-    classDef package fill: teal, stroke: black;
-```
+## <div style="display: flex; align-items: center;">ElectionGuard Python<span style="display: flex; align-items: center; margin-left: 16px"> ![Python][python-logo] <span></div>
+
+The ElectionGuard Python library is the _reference implementation_ of the ElectionGuard SDK. It covers the entire suite of functionality and processes necessary to implement an end-to-end verifiable election as part a voting system:
```

Comment:
"is the _reference implementation" <- but isn't it one of two?  Better, I think, would be "is the most complete reference implementation" or something.